src/app/pages/quickMaps/quickMaps.service.ts

```diff
@@ -65,17 +86,25 @@ export class QuickMapsService {
   public get countryId(): string {
     return this.countryIdSrc.value;
   }
+  public get countryDict(): DictionaryItem {
+    return this.countriesDictionary.getItem(this.countryId);
+  }
   public setCountryId(countryId: string, force = false): void {
-    if (force || (this.countryId !== countryId)) {
+    if (force || this.countryId !== countryId) {
       this.countryIdSrc.next(countryId);
     }
   }
 
   public get micronutrientId(): string {
     return this.micronutrientIdSrc.value;
   }
+  public get micronutrientDict(): DictionaryItem[] {
```

Comment:
This new function doesn't look like it's used.

Reply:
> Never heard of pigeon peas!

or cowpeas!

Reply:
Its used https://github.com/micronutrientsupport/micronutrient-support-tool/pull/206/files/0eeb0ac28af2e4ed9f637a793e0488a963c5e75e#diff-e263941c853363f7d21160ec7fe64e7b10ae31de2e41501f758dc2803a109f57R35